### PR TITLE
Document `install_id` and `platform_version` of app manifest endpoint

### PR DESCRIPTION
**Pull request by aamiaa**

### CODE REVIEW

File: pages/topics/client-distribution.mdx [hunk 1]
```diff
@@ -309,11 +309,13 @@ Native modules are versioned uniquely per host version.
...
 ###### Query String Params
 
-| Name     | Type   | Description                                                             |
-| -------- | ------ | ----------------------------------------------------------------------- |
-| channel  | string | The [release channel](#desktop-release-channel) to get the manifest for |
-| platform | string | The [platform](#desktop-platform-type) to get the manifest for          |
-| arch     | string | The [architecture](#desktop-architecture-type) to get the manifest for  |
+| Name              | Type   | Description                                                                 |
+| ----------------- | ------ | --------------------------------------------------------------------------- |
+| install_id?       | string | A client-generated uuid unique to the current installation                  |
```
Comment (dolfies) [inline pages/topics/client-distribution.mdx:314]:
```suggestion
| install_id?       | string | A client-generated UUID unique to the current installation                  |
```

File: pages/topics/client-distribution.mdx [hunk 2]
```diff
@@ -325,7 +327,7 @@ Native modules are versioned uniquely per host version.
 | required_modules      | array[string]                                                                 | The names of the native modules that the client requires |
 | metadata_version? ^1^ | ?integer                                                                      | The version of the manifest metadata                     |
 
-^1^ This field is only provided via the above rehosted updates endpoint. If this field is `null`, clients are expected to generate it themselves.
+^1^ This field is only provided via the above rehosted updates endpoint, and requires `install_id` and `platform_version` to be set and supported. If this field is `null`, clients are expected to generate it themselves.
 {/* TODO: How is it generated? */}
```
Comment (dolfies) [inline pages/topics/client-distribution.mdx:331]:
```suggestion
^1^ This field is only provided via the above rehosted updates endpoint, and requires `install_id` and `platform_version` to be set and supported.
```
I think we just ignore this part for now. Also, is it still only available on the updates.discord.com endpoint and not the main API?

**Review by dolfies — CHANGES_REQUESTED** [review]
very well-written!!

File: pages/topics/client-distribution.mdx (hunk 2 above)
Comment (aamiaa) [reply pages/topics/client-distribution.mdx:331]: Yep

**Pull request merged**